indico/htdocs/js/indico/modules/eventdisplay.js

```diff
@@ -23,6 +23,47 @@ $(document).ready(function() {
         });
     });
 
+    var threeRowsHeight = 69;
```

Comment:
well, that's the thing.. All the SO answers about calculating how many times the content of an element has been wrapped, say to check the `.height()`